Samples/iOS-Swift/iOS-Swift/AppDelegate.swift

```diff
@@ -8,9 +8,15 @@ class AppDelegate: UIResponder, UIApplicationDelegate {
         
         SentrySDK.start { options in
             options.dsn = "https://387714a4f3654858a6f0ff63fd551485@o447951.ingest.sentry.io/5428557"
+            
             options.beforeSend = { event in
                 return event
             }
+            
+            options.onCrashedLastRun = { eventId in
```

Comment:
`h`: What happens if there are multiple crash reports. Maybe it would be better to replace it with a property `SentrySDK.lastCrashEventId`, which is set internally. When multiple crash reports are converted to an event, this is going to be overwritten.

Reply:
A callback is not strictly necessary. The app can simply check for `SentrySDK.lastCrashEventId` at startup and if something is returned will know to prompt the user and submit the feedback for that event.

Reply:
You'd need to know the eventId somehow. `SentrySDK.lastEventId` wouldn't have the value after restart, at least not in its current form

Reply:
Understood (that's what trigger my initial request https://github.com/getsentry/sentry-cocoa/issues/699). This value needs to be exposed somehow. The fact that the sdk can now submit to the feedback api is gravy but getting the eventId is the key.

Reply:
A property won't work really, because it might take a few milliseconds until it is initialized. This happens because the SentryCrashIntegration is converting crash reports on a background thread to `SentryEvents`. So it wouldn't be really handy to wait for it to be initialized and you don't have a friendly way to achieve this. Instead, I would propose that the `onCrashedLastRun` only gets called for the first crash event. In case there are multiple crashes to send which is an edge case the others are just sent without calling the callback.

Reply:
in this case callback is the way to go and agree, multiple crash should be an edge case anyways. the main concern i had was if the callback fired more than once in the same session and it would keep prompting the user. if it fires only once for the latest crash, that's sufficient.